source/includes/_intro.md

```diff
@@ -394,25 +394,267 @@ The first step in most situations will be to instantiate a client. (TODO these c
 </tbody>
 </table>
 
-(TODO also include how to install eg with pip install?)
-
+(TODO also include how to install e.g. with pip install?)
+
+
+## Required Minimum Functionality
+
+Before your application can go live, there are a number of minimum
+implementation requirements it must meet. The test data available can help make
+sure your implementation is working as expected and can handle various types of
+errors and edge cases. To validate the functionality of your application,
+Ingresso will use the following representative tests.
+
+### List performance dates or times
+A given event may have multiple performances on different days and times, and
+the customer must be able to choose the one they want to attend. This can be
+done by listing them, or in a calendar interface, or in any other way that is
+appropriate for your application.
+
+**Representative Test:** User selects *The Unremarkable Incident of the Cat at Lunchtime*
+(event ID 7AB)<br/>
+When the event is selected in the interface,<br/>
+Then it should display the two available performances, 1 January {this year + 1}
+and 1 January {this year + 2}, at 15:30:00 UTC.
+
+### Support best available booking flow
+At a minimum, the customer must be able to specify the number of seats they
+want and for which ticket type and price band. The Ingresso backend will then
+reserve the best available seats that meet the requirements.
+
+**Representative Test:** User selects 3 tickets for event 7AB in the stalls band B<br/>
```

Comment:
I've made this change but it's worth considering that 6IF randomly generates seat IDs, so every time this test runs the results will be different - will remove references to specific seat numbers.